packages/xo-server-usage-report/src/index.js

```diff
@@ -133,6 +134,24 @@ Handlebars.registerHelper(
   value => (isFinite(value) ? round(value, 2) : '-')
 )
 
+const BYTE_CONVERSION_COEF = {
+  G: gibPower,
+  M: mibPower,
+  K: kibPower,
+}
+
+Handlebars.registerHelper(
+  'formatValue',
+  (value, unit, format) =>
```

Comment:
Is this helper used anywhere with a unit different than `B`?

Reply:
It will be used in these PRs #3463 #3463.

Reply:
I think it would be best to have a more specific `formatBytes` helper.